fake-vm-component-scripts/fake-vm/launch.py

```diff
@@ -10,25 +10,30 @@
 
 instances = []
 
-def fix_config_keys(source):
-    for k, v in source:
-        yield (k, v) if '.' not in k else (k.split('.')[1], v)
+config_keys = {
+    k: k.split('.')[1] for k in {
+        'configuration.name',
+        'configuration.address',
+        'configuration.login',
+        'configuration.failure',
```

Comment:
> we don't have component configuration

But we do - it is stored in the VM file. See `discover.py` - the name is used there.